### DISCUSSION TITLE
How do I update fields in a Content Item that has a ListPart

### DISCUSSION THREAD

**Discussion by akim-green** [OrchardCMS/OrchardCore#7267] [Q&A] [answered]
When creating a new item that contains a list part, I am able to populate fields during the initial creation. Once the item is created, all the extra fields are no longer visible and I am only able to create or remove items from the list.

How to do I modify fields that were initially populated on creation?

**Comment by deanmarcussen** [comment, marked as answer]
In the content items list find the list content item and click `Edit` instead of clicking the link the with content item title

**Reply by akim-green** [reply]
Thanks Dean.
